version.py

```diff
@@ -48,4 +48,4 @@
 #   historical commits sharing the same major version, in which case a
 #   minor version bump suffices.
 
-PROVISION_VERSION = (281, 0)  # bumped 2024-06-12 for adding firebase-admin
+PROVISION_VERSION = (282, 0)  # bumped 2024-06-16 for updating simplebar
```

Comment:
This particular upgrade is backwards-compatible, so `(281, 1)` is fine.

Reply:
Fixed [here](https://github.com/zulip/zulip/compare/042bb8d7fc80514149f2fd4ec3cdf7134fee7cac..bb4c92967ab77ed0897ab14a62fcee8c6abb17aa)